### ISSUE TITLE
Boot loop with Blackview BV9500 with multiple releases

### ISSUE THREAD

**Issue by Arteneko** [phhusson/treble_experimentations#1992]
Hey! I checked the build info and such in the Wiki, and tried to install roar, quack, and the first version that was linked in the wiki page for the BV9500 phone (tried vanilla, floss, then gapps versions).

Unfortunately, while they successfully flashed using fastboot, they are staying stuck in a boot loop (5s of "ORANGE STATE" blackview logo, black screen, 5s...).

I tried to clear the cache post-flash but that didn't help.

I saw the `/system/build.prop` fix consisting in adding the `ro.audio.ignore_effects=true` line, but I don't know how to gain a shell on the phone while it's in the stock recovery (e.g. using ADB shell) so I'm unable to try this fix as of now.

How could I proceed towards successfully flashing it, or at least successfully extracting more debug info out of the phone?

**Comment by phhusson** [comment]
You flashed arm64 ab variant?

ignore_effects issue wouldn't manifest this way, it would stay stuck on
android bootanimation

**Comment by Arteneko** [comment]
I flashed the arm64 A variant, as suggested in the wiki page.

A few months ago, I flashed the AB version by error and bricked the motherboard, so I would probably avoid doing it again

**Comment by phhusson** [comment]
It was released with Android 9? If so, then it's definitely AB.

**Comment by Arteneko** [comment]
Should I use the quack AB version then? Or the AOSP 9 version.

(And that would then conflict with what the wiki page says, so I'm surprised)

**Comment by Arteneko** [comment]
I just read this paragraph that was sitting just above the phone list.

> Why should I use GSI for A/B devices to some A-only devices? : On Android 9, Google changed the requirements and all devices shipped with Android 9 must use "system-as-root". Before this change, only A/B devices were system-as-root and GSI name was separated by A-only and A/B. For this historical reasons, you have to use GSI for A/B on the devices shipped with Android 9+..

That makes sense. I'm still afraid of doing this operation, but will try anyways.

**Comment by Arteneko** [comment]
Indeed, it is as you said.

As [the wiki page for the BV9500](https://github.com/phhusson/treble_experimentations/wiki/Blackview-BV9500) documents, it appears to be a phone needing VNDK support, so I picked the Roar A/B arm64 with vndklite, and my phone booted right away.

Thank you for your time and your work, I have learned a bit along the way and thoroughly documented my steps and analysis as to not make this mistake again.
